.circleci/config.yml

```diff
@@ -132,6 +129,8 @@ jobs: # a collection of steps
     docker:
       - image: circleci/node:12.16.2
       # TODO get the sdk build from saved workspace
+    environment:
+      PUBLIC_URL: .
```

Comment:
I don't know what it is ^^. Could you tell me what is it for?

Reply:
This environment variable used by create-react-app and it appends it to each link for JS and CSS files. See here: https://create-react-app.dev/docs/advanced-configuration/

The problem is that create-react-app assumes that the App is hosted on the root of the domain, but ours will not be on the root (it will be under `https://rakutentech.github.io/js-miniapp/sample`) . So I set this to `.` so that links to assets will be like `./script.js`.